core/model/src/main/java/org/eclipse/sw360/antenna/model/license/FromXmlLicenseInformationBuilder.java

```diff
@@ -114,7 +114,7 @@ public void setException(LicenseBuilder exception) {
 
         @Override
         public LicenseInformation build() {
-            return new WithLicense((License) license.build(), (License) exception.build());
+            return new WithLicense(license.id, license.commonName, license.text, exception.id, exception.commonName, exception.text);
```

Comment:
A general remark: We should watch out to not re-implement too much of either https://github.com/oss-review-toolkit/ort/tree/master/spdx-utils/src/main/kotlin or https://github.com/spdx/Spdx-Java-Library/tree/master/src/main/java/org/spdx/library/model/license here.

Reply:
In principle I agree, but there has to be a step-by-step move towards the intended solution. In the meanwhile, we need this step to continue with "smaller" steps.